openapi.yml

```diff
@@ -1049,6 +1049,14 @@ components:
           $ref: '#/components/schemas/LanguageTag'
         use:
           $ref: '#/components/schemas/FileUse'
+        sdrGeneratedText:
+          description: Indicates if the text (OCR/captioning) was generated by SDR.
+          type: boolean
+          default: false
+        correctedForAccessibility:
+          description: Indicates if text that has been verified for accessibility/correctness.
+          type: boolean
+          default: false
```

Comment:
Should these also properties of the `RequestFile` model? Currently, I believe the two models have the same properties (main diff is the external identifier is optional on the request flavor).

Reply:
Yes, i guess so - I'll add it there too.  I can't say I know when you'd use one or the other though, do you know?